.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.lock
 .DS_Store
 docs
+.idea
```

Comment:
please remove and add to your global .gitignore for this is not project specific, more of a personal ignore.